public_dropin_gpu_environments/nim_llm/dr_requirements.txt

```diff
@@ -43,7 +46,7 @@ datarobot==3.3.1
     # via
     #   -r dr_requirements.in
     #   datarobot-drum
-datarobot-drum==1.11.5
+datarobot-drum @ git+https://github.com/datarobot/datarobot-user-models.git@ec4587c0a09024deac1545f2460411be90a5794a#subdirectory=custom_model_runner&egg=datarobot-drum
```

Comment:
Will remove this once this PR merges to master and I can release the version of DRUM